app/views/shinycms/discussions/_comment_posted_by.html.erb

```diff
@@ -0,0 +1,14 @@
+Posted by
+<% if comment.author_type == 'user' %>
+  <%= user_profile_link( comment.author ) %>
+<% elsif comment.author_type == 'pseudonymous' %>
+  <% if comment.author_url.present? %>
+    <%= link_to comment.author_name, comment.author_url %>
```

Comment:
Consider adding `rel='nofollow'` or `rel='ugc'` to author links to reduce incentive for spammers: https://support.google.com/webmasters/answer/96569?hl=en